ext/hal/silabs/gecko/emlib/inc/em_ramfunc.h

```diff
@@ -143,7 +143,7 @@ extern "C" {
 
 #elif defined(__GNUC__)
 /* Simplicity Studio, Atollic and Vanilla armgcc */
-#define SL_RAMFUNC_DECLARATOR          __attribute__ ((section(".ram")))
+#define SL_RAMFUNC_DECLARATOR          __attribute__ ((section(".ramfunc")))
```

Comment:
I preserved the method used by SiLabs to keep modifications to minimum. Zephyr is using a different approach: `__attribute__((long_call, section(".ramfunc")))`. Without the `long_call` keyword the linker will create the so called veneers in case the range of the branch instruction is insufficient. With the `long_call` keyword compiler will always use the long jump. BL instruction on processors that support Thumb-2 technology has a range of 16MB.